test/timing_load_creds.c

```diff
@@ -76,11 +77,12 @@ static void usage(void)
     fprintf(stderr, "          p for private key\n");
     exit(EXIT_FAILURE);
 }
+# endif
 #endif
 
 int main(int ac, char **av)
 {
-#ifdef OPENSSL_SYS_UNIX
+#if defined(_POSIX_VERSION) && _POSIX_VERSION >= 200112L
```

Comment:
The `#else` part should probably be modified, something like this perhaps?

``` C
#else
    fprintf(stderr, "This tool is not supported on this platform for lack of POSIX1.2001 support\n");
    exit(EXIT_FAILURE);
#endif
```

Reply:
... you missed a spot :wink:

Remove the Windows and VMS specific bits, they're quite unnecessary, really

Reply:
OK, makes some sense too.